bcda/auth/provider.go

```diff
@@ -84,10 +84,6 @@ type Provider interface {
 	// RevokeAccessToken a specific access token identified in a base64 encoded token string
 	RevokeAccessToken(tokenString string) error
 
-	// TODO refactor input to be AuthData
-	// AuthorizeAccess asserts that a base64 encoded token string is valid for accessing the BCDA API
-	AuthorizeAccess(tokenString string) error
```

Comment:
this is removed, per the merge/refactor here, correct?

https://github.com/CMSgov/bcda-app/pull/841/files#r1180735589

Reply:
Yes that's correct. This is removed from the SSAS/auth provider and is now just a helper function that lives directly in the middleware.